### ISSUE TITLE
Voice Over Improvements suggestions. 

### ISSUE THREAD

**Issue by rezzap** [Automattic/simplenote-electron#2471]
On Windows, using NVDA:

When you press the New Note button, you're taken inside the correct field, but the label is wrong, it announces "search notes all notes". I don't really think the field needs a label. What does need a label is the search button under the search
field, which currently doesn't have one. Also, arrowing down from that search button, NVDA reads "table with zero rows and zero columns grid", I don't know what that is but if that random info could somehow be
hidden from the screen-reader that'd be great.

Reported in 3487018-zen (user is willing to help with beta testing if needed)

**Comment by azollafox** [comment]
Windows Version    10.0.18363 Build 18363
NVDA Version: alpha-21348,6af30832
